locales/en/yourloops.json

```diff
@@ -266,6 +267,7 @@
   "notification-patient-invitation-by-team": "You're invited to share your diabetes data with <strong>{{careteam}}</strong>.",
   "notification-patient-invitation-wrong-code": "The team code does not match the one of {{careteam}}.",
   "open-team-menu": "Open team menu",
+  "open-training": "Open {{training}}",
```

Comment:
I don't figure out why adding a variable training ? Is the text subject to change ?

Reply:
you're right it's always training I think